src/std/make.ss

```diff
@@ -225,9 +233,11 @@
          ([ssi: mod . opts]
           (lp rest (cons (file-e mod ".ssi") files)))
          (else
+          ; otherwise it's static-include: or copy:, no deps,
+          ; not relevant for making a depgraph
           (lp rest files))))
       (else
-       (make-depgraph (reverse files))))))
+       (reverse files)))))
```

Comment:
Can you explain me what happened here?

Reply:
I split `make-depgraph/spec` into two function:
 - `buildspec-depfiles` which is what most of the old body turned into
 - `make-depgraph` which already existed

Where the new definition of `make-depgraph/spec` is the composition of the two. The change is the difference between `make-depgraph/spec` and `buildspec-depfiles`.